src/main/java/ua/nure/makestart/config/EncriptionConfig.java

```diff
@@ -0,0 +1,14 @@
+package ua.nure.makestart.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class EncriptionConfig {
+    @Bean
+    public PasswordEncoder getPasswordEncoder() {
+        return new BCryptPasswordEncoder(8);
```

Comment:
Don't use magic number, what does it mean? Add in properties or create final value